### PR TITLE
please add mkdocs site materials

### PR DESCRIPTION
**Pull request by morganem0**
I added content for  generating the documentation site into the docs/ folder, and added the mkdocs site/ folder into .gitignore. Also added an example data folder for the user tutorial. 
* docs/ has markdown pages for generating the site
* mkdocs.yml provides configuration for mkdocs to generate the site pages
* ll-example folder has materials for the "basic usage tutorial" in quickstart.md

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+* Validate metadata for registered files
+* Run a Preservation action on example data files
+* Alter an example data file to cause an error; catch the error by re-running the Preservation action
+* Re-register the altered file, and preserve the file in its new, altered state
+
+### System Requirements for this tutorial
+Longleaf scripts rely on common UNIX programs. In Mac OS X and Linux operating systems, these programs will likely already be installed, but some of these tools, such as `rsync` may be missing from a Windows system unless you have installed it.
+
+### Example data directory for this tutorial
+The example data directory for this tutorial is named 'll-example'; it is located (at the top level within this repository).
```
Comment (cazzerson) [inline docs/quickstart.md]: This either needs to be changed or ll-example needs to be moved to the top level. Thoughts?

File: docs/mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+site_name: Longleaf Documentation
```
Comment (cazzerson) [inline docs/mkdocs.yml]: mkdocs.yml should be at the root of the repo

File: docs/quickstart.md (hunk 1 above)
Comment (morganem0) [reply docs/quickstart.md]: I'm fine with either. Do you have a preference for where 'll-example/' should go?

File: docs/mkdocs.yml (hunk 2 above)
Comment (morganem0) [reply docs/mkdocs.yml]: Done.

File: docs/quickstart.md (hunk 1 above)
Comment (morganem0) [reply docs/quickstart.md]: After discussing, we decided to leave ll-example in docs/. I updated the wording in quickstart.md so it correctly describes the location of the example data directory.

**Pull request merged**